manifests/sql.pp

```diff
@@ -2,7 +2,7 @@
 define freeradius::sql (
   Enum['mysql', 'mssql', 'oracle', 'postgresql'] $database,
   Freeradius::Password $password,
-  Variant[Stdlib::Host, Stdlib::IP::Address] $server                                = 'localhost',
+  Variant[Stdlib::Host, Stdlib::IP::Address::Nosubnet] $server                      = 'localhost',
```

Comment:
```suggestion
  Variant[Stdlib::Host, Stdlib::IP::Address] $server                                = 'localhost',
```

(actually it turns out that this can just be Stdlib::Host - as Host already covers single IP addresses which is nice - but I don't think we should make that change in this PR)